androidsdk/src/main/java/com/wootric/androidsdk/views/phone/SurveyLayoutPhone.java

```diff
@@ -336,7 +332,6 @@ private void setColors() {
         mColorEnabled = res.getColor(mSettings.getSurveyColor());
         mRatingBar.setSelectedColor(mColorSelected);
 
-        mBtnEditScore.setTextColor(mColorEnabled);
```

Comment:
@diegoserranoa I think you were referring this line of code yeah. Here what I think I can do more is set Background Color like `mBtnEditScore.setBackgroundColor(mColorEnabled);`